### ISSUE TITLE
[Intro]: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds for 'Code' part in 'Intro' page.[Sev2][WCAG1.4.3][Auto]

### ISSUE THREAD

**Issue by SrujanaAnaganti** [microsoft/tsdoc.org-website#11]
**Test Environment:**
OS Build: Windows 10
Browser: Edge Dev
URL: https://tsdoc.org/

**Repro Steps:**
1) Open the above URL and login with valid credentials.
2) TSDoc page will get displayed.
3) Navigate to 'Intro' link and activate it.
4) Now run Accessibility Insights for web tool and observe the issue.

**Actual:**
Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds for 'Code' part in 'Intro' page. These elements have insufficient contrast ratio of 3.99.

**Expected:**
Contrast ratio for the text must be greater than or equal to 4.5:1

**User Impact:**
When a minimum contrast ratio is not provided between text and its background, it is difficult for low-vision users as well as anyone in extreme lighting environments to read the content.

**Note:** Same issue is repro through out the URL where ever coding part is exist.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum
<img width="960" alt="Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds for 'Code' part in 'Intro' page" src="https://user-images.githubusercontent.com/85154417/121009489-dadd3100-c7b1-11eb-86fe-d62535ed3355.png">
